### PR TITLE
#17: Added rendering for aggregate functions `GROUPING` and `APPROXIMATE COUNT DISTINCT`

### LINKED ISSUE #17: Render basic aggregate functions
## Acceptance Criteria

Renders the following [aggregate functions](https://docs.exasol.com/db/latest/sql_references/functions/aggregatefunctions.htm)

- [x] `APPROXIMATE_COUNT_DISTINCT`
- [x] `GROUPING` / `GROUPING_ID`

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #17.

### CODE REVIEW

File: spec/queryrenderer/literal_constructors.lua [hunk 1]
```diff
@@ -44,4 +44,30 @@ function literal_constructors.interval_ds(value, precision, fraction)
     }
 end
 
+--- Wrap Lua literals so that they look like the literal definition the Virtual Schema API uses.
+-- When fed with Lua literals, the function wraps the parameter in the corresponding VS API definition. Tables remain
+-- unchanged.
+-- @param ... list of Lua literals and or tables
```
Comment (tomuben) [inline spec/queryrenderer/literal_constructors.lua:50]:
```suggestion
-- @param ... list of Lua literals and/or tables
```

**Review by tomuben — COMMENTED** [review]
just a tiny suggestion

File: doc/changes/changes_2.2.0.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ 
+## Summary
+
+Release 2.2.0 of `virtual-schema-common-lua` adds SQL rendering for aggregate functions.
+We also removed the standard dependency check, since it only targets Java dependencies and this project is a pure Lua project.
+
+## Features
+
+* #17: Added rendering for aggregate functions `GROUPING` and `APPROXIMATE COUNT DISTINCT`
+* #47: Removed dependency check CI build
```
Comment (kaklakariada) [inline doc/changes/changes_2.2.0.md:13]: This is not done in this PR?

File: spec/QueryRenderer_spec.lua [hunk 3]
```diff
@@ -11,4 +11,26 @@ describe("QueryRenderer", function()
...
+            selectList = {
+                {
+                    type = "function_aggregate",
+                    name = "approximate_count_distinct",
+                    arguments = {
+                        {
+                            type = "column",
+                            name = "C1",
+                            columnNr = 2,
+                            tableName ="T2"
```
Comment (kaklakariada) [inline spec/QueryRenderer_spec.lua:27]:
```suggestion
                            tableName = "T2"
```

File: spec/queryrenderer/AggregateFunctionAppender_spec.lua [hunk 4]
```diff
@@ -0,0 +1,46 @@
+package.path = "src/?.lua;" .. package.path
+require("busted.runner")()
+local literal = require("queryrenderer.literal_constructors")
+local Query = require("exasolvs.Query")
+local AggregateFunctionAppender = require("exasolvs.queryrenderer.AggregateFunctionAppender")
+
+function it_asserts(expected, actual, explanation)
```
Comment (kaklakariada) [inline spec/queryrenderer/AggregateFunctionAppender_spec.lua:7]: Global function by intention?
```suggestion
local function it_asserts(expected, actual, explanation)
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]

**Review by kaklakariada — DISMISSED** [review]

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**